tools/test/spm/test_generate_partition_code.py

```diff
@@ -23,7 +23,7 @@
 import pytest
 from jinja2.defaults import DEFAULT_FILTERS
 
-from test_data import *
+from tools.test.spm.test_data import *
```

Comment:
@ARMmbed/mbed-os-tools @orenc17 

I feel like this isn't the correct way to fix this import issue, but it works.
Open to better suggestions.

Reply:
Perhaps: 
```suggestion
from .test_data import *
```

Reply:
looks fine, but if it aint broken why fix it?

Reply:
@orenc17 : The commit msg: `Py3 tests were not running as a result`

Reply:
> looks fine, but if it aint broken why fix it? 

It did break, in Py35 iirc.